### PR TITLE
WS1 - Static Recipe - davidhdzara

### PR DESCRIPTION
**Pull request by davidhdzara**
### Acceptance Criteria

- [x] You must use good [semantics in HTML tags](https://es.semrush.com/blog/html-semantico/).
- [x] You have to import fonts from a CDN or create a font from 0 with [@font-face](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face).
- [x] Use [SASS preprocessor](https://sass-lang.com/documentation/style-rules/) or others ([Less vs Sass vs Stylus](https://stackshare.io/stackups/less-vs-sass-vs-stylus)).
- [x] Use [variables](https://sass-lang.com/documentation/variables/) for style sheets without hard code.
- [x] You have to [reset the CSS](https://www.aluracursos.com/blog/reset-css-que-es-ejemplos-como-crear-y-utilizar) so that you can work with the HTML elements in a clean way.
- [x] You have to use properties of [box model](https://web.dev/i18n/es/learn/css/box-model/) such as box-sizing, margin, padding, and border.
- [x] You have to use CSS display layout properties:
  - [CSS Inline vs Inline-Block vs Block](https://www.samanthaming.com/pictorials/css-inline-vs-inlineblock-vs-block/)
  - [Grid & flexbox MDN](https://developer.mozilla.org/es/docs/Web/CSS/CSS_grid_layout/Relationship_of_grid_layout_with_other_layout_methods#grid_y_flexbox)
- [x] You have to use CSS [positioning](https://css-tricks.com/almanac/properties/p/position/) and [z-index](https://css-tricks.com/css-painting-order/) properties to place the overlapping elements.

### Demo
image

### Comentarios Taller:

Felicidades todos los criterios de aceptación completos:
**Nota:** `4.7 de 5.0`

> Termina los comentarios faltantes, así mejorar la nota

### CODE REVIEW

**Comment by DanteCely** [comment]
Te falta implementar SASS 🙄

File: davidhdzara1/1-cooking-recipe-html-css/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+h1 {
+  color: #072b30;
+ margin-top: 0;
+ float: right;
+
+}
+
+h1 span {
+  font-family: "Kaushan Script", cursive;
+  display: block; /* !Permite que se genere un bloqueo */
```
Comment (DanteCely) [inline davidhdzara1/1-cooking-recipe-html-css/css/styles.css:55]: Este permite que la etiqueta se comporte como un bloque completamente horizontal en la pantalla 
[CSS Inline vs Inline-Block vs Block](https://www.samanthaming.com/pictorials/css-inline-vs-inlineblock-vs-block/)

File: davidhdzara1/1-cooking-recipe-html-css/index.html [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/normalyze.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+  </head>
+
+  <body>
+    <header>
+      <img src="./img/logo.jpg" alt="" class="logo" />
+      <h1><span>Platos</span> FRÍOS</h1>
```
Comment (DanteCely) [inline davidhdzara1/1-cooking-recipe-html-css/index.html:20]: No debes usar titulos completamente en Mayúsculas porque el navegador lo entenderá como siglas. En lugar de eso utiliza css para agregar las Mayúsculas [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)

File: davidhdzara1/1-cooking-recipe-html-css/index.html [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+    <main class="maininformation">
+      <aside class="leftcontent">
+        <article class="left-content">
+          <h3 class="ingredientes">Ingredientes</h3>
+
+          <ul>
+            <li>500g de langostinos de nuestra oferta</li>
+            <li>1 bolsa de lechugas variadas gourmet MAS</li>
+            <li>1 bote de maíz ELIGES</li>
```
Comment (DanteCely) [inline davidhdzara1/1-cooking-recipe-html-css/index.html:32]: NO puedes usar MAyusculas, cambia a mayusculas con estilos

File: davidhdzara1/1-cooking-recipe-html-css/index.html [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+              >
+                <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                <path
+                  d="M12 3c1.918 0 3.52 1.35 3.91 3.151a4 4 0 0 1 2.09 7.723l0 7.126h-12v-7.126a4 4 0 1 1 2.092 -7.723a4 4 0 0 1 3.908 -3.151z"
+                />
+                <path d="M6.161 17.009l11.839 -.009" />
+              </svg>
+              Facil
+            </span>
+          </div>
```
Comment (DanteCely) [inline davidhdzara1/1-cooking-recipe-html-css/index.html:122]: A modo de sugerencia utiliza iconos de Google Icons, más fácil de mantener los íconos. https://fonts.google.com/icons

File: davidhdzara1/1-cooking-recipe-html-css/sass/style.scss [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+h1 span {
+  font-family: "Kaushan Script", cursive;
+  display: block; /* !Permite que se genere un bloqueo */
+  font-size: 5rem;
+}
+
+.titulo-plato {
+  background-color: $color-titulos;
+  color: white;
```
Comment (DanteCely) [inline davidhdzara1/1-cooking-recipe-html-css/sass/style.scss:50]: Usa una variable para white como $white

**Review by DanteCely — APPROVED** [review]

File: davidhdzara1/1-cooking-recipe-html-css/index.html (hunk 2 above)
Comment (davidhdzara) [reply davidhdzara1/1-cooking-recipe-html-css/index.html:20]: Se realiza cambio de codigo CSS y se establece  text-transform: uppercase; sobre el H1 correspondiente

File: davidhdzara1/1-cooking-recipe-html-css/sass/style.scss (hunk 5 above)
Comment (davidhdzara) [reply davidhdzara1/1-cooking-recipe-html-css/sass/style.scss:50]: Se realiza el cambio utilizando la variable $color-blanco;

**Pull request merged**